### PR TITLE
:book: Update swagger OpenAPI spec

### PR DESCRIPTION
**Pull request by aufi**
Updating swagger files to be up-to-date 2023-03-02 including changes from:
- https://github.com/konveyor/tackle2-hub/pull/226
- https://github.com/konveyor/tackle2-hub/pull/245
- https://github.com/konveyor/tackle2-hub/pull/246
- https://github.com/konveyor/tackle2-hub/pull/249

### CODE REVIEW

File: api/file.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func (h FileHandler) Create(ctx *gin.Context) {
 // @summary Get a file by ID.
 // @description Get a file by ID. Returns api.File when Accept=application/json else the file content.
 // @tags get
-// @produce json octet-stream
+// @produce octet-stream
```
Comment (aufi) [inline api/file.go:127]: The swagger generator supports single produce type only.

File: api/setting.go [hunk 2]
```diff
@@ -42,7 +42,7 @@ func (h SettingHandler) AddRoutes(e *gin.Engine) {
 // @description Get a setting by its key.
 // @tags get, setting
 // @produce json
-// @success 200 {object} interface{}
+// @success 200 {object} api.Setting
```
Comment (aufi) [inline api/setting.go:45]: ```{object}``` or ```interface{}``` are not supported types for the generator, more details: https://swagger.io/docs/specification/data-models/data-types/

**Comment by aufi** [comment]
Some features of the API are not suported by the swagger generator (added comments to the source code lines). Some godoc lines were updated to cover at least part of the described API function to allow OpenAPIdocs generator parse it.

**Review by jortel — APPROVED** [review]
LGTM

**Comment by jortel** [comment]
Can you: `make docs-binding` in the PR as well?

**Pull request merged**